crates/matrix-sdk/src/client/builder.rs

```diff
@@ -323,11 +323,21 @@ impl ClientBuilder {
         self
     }
 
+    /// Don't trust any system root certificates, only trust the certificates
+    /// provided through
+    /// [`add_root_certificates`][ClientBuilder::add_root_certificates].
+    #[cfg(not(target_arch = "wasm32"))]
+    pub fn disable_built_in_root_certificates(mut self) -> Self {
+        self.http_settings().disable_built_in_root_certificates = true;
+        self
+    }
+
     /// Specify a [`reqwest::Client`] instance to handle sending requests and
     /// receiving responses.
     ///
     /// This method is mutually exclusive with [`proxy()`][Self::proxy],
-    /// [`disable_ssl_verification`][Self::disable_ssl_verification] and
+    /// [`disable_ssl_verification`][Self::disable_ssl_verification],
+    /// [`add_root_certificates`][Self::add_root_certificates] and
```

Comment:
```suggestion
    /// [`add_root_certificates`][ClientBuilder::add_root_certificates] and
```

Reply:
Ah yeah, have updated them all in that comment.